README.md

```diff
@@ -4,6 +4,14 @@ Extended documentation for the [Botan](https://botan.randombit.net/)
 cryptography library in the context of the BSI project "Pflege und
 Weiterentwicklung der Kryptobibliothek Botan".
 
+## Repository Anatomy
+
+This monorepo contains documents (in `/docs`) as well as auxiliary helper
+scripts (in `/tools`). Most documents come with a `Makefile` to build them as
+PDF, HTML or other formats. Some generators depend on the helper scripts in
+`/tools`. We use [Poetry](https://python-poetry.org/) to manage the dependencies
+local and external dependencies.
```

Comment:
Wording (probably drop the `dependencies` in line 12)

Reply:
Yeah. Dependency-hell.